ckanext/pigma_theme/plugin.py

```diff
@@ -23,7 +23,6 @@ class Pigma_ThemePlugin(plugins.SingletonPlugin):
     def update_config(self, config_):
         toolkit.add_template_directory(config_, 'templates')
         toolkit.add_public_directory(config_, 'public')
-        toolkit.add_resource('fanstatic', 'pigma_theme')
```

Comment:
I think that will break link to images in that folder.

Reply:
I'll checkout.

Reply:
I guess you don't have locally the groups using them